src/safeds/data/tabular/plotting/_table_plotter.py

```diff
@@ -454,6 +456,88 @@ def moving_average_plot(self, x_name: str, y_name: str, window_size: int) -> Ima
 
         return _figure_to_image(fig)
 
+    def histogram_2d(
+        self,
+        x_name: str,
+        y_name: str,
+        *,
+        x_max_bin_count: int = 10,
+        y_max_bin_count: int = 10,
+        theme: Literal["dark", "light"] = "light",
+    ) -> Image:
+        """
+        Create a 2D histogram for two columns in the table.
+
+        Parameters
+        ----------
+        x_name:
+            The name of the column to be plotted on the x-axis.
+        y_name:
+            The name of the column to be plotted on the y-axis.
+        x_max_bin_count:
+            The maximum number of bins to use in the histogram for the x-axis. Default is 10.
+        y_max_bin_count:
+            The maximum number of bins to use in the histogram for the y-axis. Default is 10.
+
+        Returns
+        -------
+        plot:
+            The plot as an image.
+
+        Raises
+        ------
+        ColumnNotFoundError
+            If a column does not exist.
+        OutOfBoundsError:
+            If the actual value is outside its expected range (x_max_bin_count, y_max_bin_count).
```

Comment:
The description is not fitting: This error is raised if `x_max_bin_count` or `y_max_bin_count` are < 1.